service/src/main/kotlin/fi/espoo/evaka/shared/security/Action.kt

```diff
@@ -1176,9 +1138,21 @@ sealed interface Action {
 
         override fun toString(): String = "${javaClass.name}.$name"
     }
-    enum class MessageDraft(override vararg val defaultRules: ScopedActionRule<in MessageDraftId>) :
-        ScopedAction<MessageDraftId> {
-        UPLOAD_ATTACHMENT(IsEmployee.hasPermissionForMessageDraft());
+    enum class MessageAccount(
+        override vararg val defaultRules: ScopedActionRule<in MessageAccountId>
+    ) : ScopedAction<MessageAccountId> {
+        ACCESS(
+            IsEmployee.hasPersonalMessageAccount(),
+            HasUnitRole(UNIT_SUPERVISOR, SPECIAL_EDUCATION_TEACHER).hasDaycareMessageAccount(),
+            IsEmployee.hasDaycareGroupMessageAccount(),
+            IsEmployee.hasMunicipalMessageAccount(),
+            IsMobile(requirePinLogin = true).hasPersonalMessageAccount(),
+            IsMobile(requirePinLogin = true)
+                .hasDaycareMessageAccount(UNIT_SUPERVISOR, SPECIAL_EDUCATION_TEACHER),
+            IsMobile(requirePinLogin = true).hasDaycareGroupMessageAccount(),
+            IsMobile(requirePinLogin = true).hasMunicipalMessageAccount(),
```

Comment:
I don't think this rule makes much sense so I think it should be removed. It's not even practically possible to send municipal messages from employee mobile.